docs/rtd_environment.yml

```diff
@@ -38,3 +38,4 @@ dependencies:
       - types-openpyxl  # dev, tests
       - types-jinja2  # dev, tests
       - sqlalchemy-stubs  # dev, tests
```

Comment:
Need to add nlopt, code example [errors out](https://optimagic--536.org.readthedocs.build/en/536/how_to/how_to_algorithm_selection.html#step-2-experiments).

Reply:
Done, thanks!